### DISCUSSION TITLE
"Add a DNS TXT record" option not available

### DISCUSSION THREAD

**Discussion by dustybusby** [community/community#144466] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

I'm trying to add my custom domain, per the instructions here: https://docs.github.com/en/pages/configuring-a-custom-domain-for-your-github-pages-site/verifying-your-custom-domain-for-github-pages. I should get an "Add a DNS TXT record" option (Step #5). However, there is no such option on my Pages settings after I add my custom domain and it is trying to validate DNS immediately (and failing as expected). How can I get the proper DNS records that need to be created with this step missing?

**Comment by Jemjeqt** [comment, marked as answer]
Here's just the concise conclusion in text:
To resolve the missing DNS TXT record option issue for GitHub Pages custom domain setup:
Remove your current custom domain entry, wait a few minutes, then add it again. Meanwhile, set up these essential DNS records with your domain provider:
A Records pointing to:
185.199.108.153
185.199.109.153
185.199.110.153
185.199.111.153
And a CNAME Record:
www → your-username.github.io
Once these are set up, GitHub Pages should validate your domain successfully. If issues persist, contact GitHub Support for assistance.

**Reply by dustybusby** [reply]
Thanks. What is the host name for the A records?

**Reply by dustybusby** [reply]
fyi...got it working. Used the @ as Host for the A record.

**Comment by mecodeatlas** [comment]
Hi @dustybusby 👋🏾,It looks like @Jemjeqt's response was the answer to your question here. Can you please mark their response as the answer? When you mark a question as an answer, GitHub will highlight the comment and replies to the comment to help visitors quickly find the answer.
![image](https://github.com/user-attachments/assets/b89e7161-c2d1-47e2-a058-242df633e6b2)
